pyproject.toml

```diff
@@ -19,8 +19,10 @@ dependencies = [
     "gwcs >=0.19.0",
     "numpy >=1.24",
     "astropy >=5.3.0",
+    # "rad >=0.22.0",
     # "rad >=0.23.0, <0.24.0",
-    "rad @ git+https://github.com/spacetelescope/rad.git",
+    #"rad @ git+https://github.com/spacetelescope/rad.git",
+    "rad @ git+https://github.com/spacetelescope/rad.git@skycell",
```

Comment:
```suggestion
    "rad @ git+https://github.com/perrygreenfield/rad.git@skycell2",
```

Reply:
@perrygreenfield would you update this pin so it points to the branch for https://github.com/spacetelescope/rad/pull/536 (and we can remove the skycell branch on the spacetelescope fork).